docs/supported-version.md

```diff
@@ -0,0 +1,15 @@
+# Xen Orchestra compatibility
+
+Xen Orchestra is designed to work exclusively on XCP-ng and Citrix XenServer hypervisor. Theorically, Xen Orchestra is fully functionnal with any version of these two hypervisor. However, to benefit from the best support quality, our product is tested to work the best with the following versions:
+
+## Citrix XenServer (Citrix hypervisor)
```

Comment:
We also need to provide a list of XSO's for each affected XS version

Reply:
XSO?

Reply:
* VDI I/O error affecting XS 7.5: https://bugs.xenserver.org/browse/XSO-873
* VHD >1TiB affecting all versions of XS and XCP-ng before 7.5: https://bugs.xenserver.org/browse/XSO-868